ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -127,11 +127,15 @@
                   :validateTransfer="validate"
                   @isValidTransferOwners="isValidTransferOwners = $event"
                 />
-                <TransferDetails v-if="showTransferDetails" ref="transferDetailsComponent" />
+                <TransferDetails
+                  v-if="hasUnsavedChanges"
+                  ref="transferDetailsComponent"
+                  @isValid="isTransferDetailsFormValid = $event"
```

Comment:
I see, so you refactored to call the validation and then emit back the result, instead of returning results... I'm ok with that.

Reply:
Definitely! A small change.
Just helps the parent component (MhrInfo) remain reactive to the childs validation state.